deploy/lib/server_config.rb

```diff
@@ -2685,8 +2685,9 @@ def ServerConfig.properties(prop_file_location = @@path)
     properties = ServerConfig.load_properties(default_properties_file, "ml.")
     properties.merge!(ServerConfig.load_properties(properties_file, "ml."))
 
+    #Look for optional shared_config, if it is set grab the properties from path relative to the root of the roxy project
     if properties['ml.shared_config'] then
```

Comment:
I'm thinking we're also going to need to do these tests :

1) Does a shared properties file exist? (Could aid debugging)
2) Should a shared properties file be used for this environment? My understanding of your use case is that there's inevitably going to end up being two shared properties files applied to different environments respectively. So we'll need a mechanism to map environment properties to shared properties appropriately.